### PR TITLE
add create workflow

### PR DESCRIPTION
**Pull request by cooper-lzy**
### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: docs-2.0/nebula-explorer/workflow/2.create-workflow.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  ![connect-component](https://docs-cdn.nebula-graph.com.cn/figures/ex-connect-220623-cn.png)
+
+5. Click the graph computing component and set the following parameters in the configuration panel on the right side.
+
+  ![connect-component](https://docs-cdn.nebula-graph.com.cn/figures/ex-algorithm-220623-en.png)
+
+  |Parameters|Description|
+  |:---|:---|
+  |PageRank|Click ![pencil](https://docs-cdn.nebula-graph.com.cn/figures/workflow-edit.png) to modify the component name to identify the component.|
+  |Input| Three data sources are supported as input. <br>**Nebula Graph**: Users must select the graph space and corresponding edge types.<br>**Dependence**: The system will automatically recognize  the data source according to the connection of the anchor.<br>**HDFS**: Users must select HDFS and fill in the relative path of the data source file.|
```
Comment (abby-cyber) [inline docs-2.0/nebula-explorer/workflow/2.create-workflow.md:46]:
```suggestion
  |Input| Three data sources are supported as input. <br>**Nebula Graph**: Users must select one graph space and corresponding edge types.<br>**Dependence**: The system will automatically recognize the data source according to the connection of the anchor.<br>**HDFS**: Users must select HDFS and fill in the relative path of the data source file.|
```

File: docs-2.0/nebula-explorer/workflow/2.create-workflow.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  |Parameter settings| Set the parameters of the graph algorithm. The parameters of different algorithms are different. Some parameters can be obtained from any upstream component where the anchor are shown in yellow.|
+  |Output| Display the column name of the graph computing results. The name can not be modified.|
+  |Execution settings| **Machine num**: The number of machines executing the algorithm.<br>**Processes**: The total number of processes executing the algorithm. Allocate these processes equally to each machine based on the number of machines.<br>**Threads**: How many threads are started per process.|
+  |Results| Set the result saved in HDFS or Nebula Graph.<br>**HDFS**: The save path is automatically generated based on the job and task ID.<br>**Nebula Graph**: Tags need to be created beforehand in the corresponding graph space to store the results. For more information about the properties of the tag, see [Algorithm overview](../../graph-computing/algorithm-description.md).<br>Some algorithms can only be saved in the HDFS.|
```
Comment (abby-cyber) [inline docs-2.0/nebula-explorer/workflow/2.create-workflow.md:50]: Set the result saved in HDFS or Nebula Graph.

Set the restoration path of the results in HDFS or Nebula Graph?

File: docs-2.0/nebula-explorer/workflow/2.create-workflow.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+6. Click ![pencil](https://docs-cdn.nebula-graph.com.cn/figures/workflow-edit.png) next to the automatically generated workflow name  at the upper left corner of the canvas page to modify the workflow name, and click **Run** at the upper right corner of the canvas page. The job page is automatically displayed to show the job progress. You can view the result after the job is completed. For details, see [Job management](4.jobs-management.md).
```
Comment (abby-cyber) [inline docs-2.0/nebula-explorer/workflow/2.create-workflow.md:52]:
```suggestion
6. Click ![pencil](https://docs-cdn.nebula-graph.com.cn/figures/workflow-edit.png) next to the automatically generated workflow name at the upper left corner of the canvas page to modify the workflow name, and click **Run** at the upper right corner of the canvas page. The job page is automatically displayed to show the job progress. You can view the result after the job is completed. For details, see [Job management](4.jobs-management.md).
```

File: docs-2.0/nebula-explorer/workflow/2.create-workflow.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+  !!! note
+
+        When you click **Run**, the workflow will be automatically saved. If you do not perform graph computing and only modify, click ![save](https://docs-cdn.nebula-graph.com.cn/figures/workflow-save-220623.png) to save the modification, or click ![save](https://docs-cdn.nebula-graph.com.cn/figures/workflow-saveAs-220623.png) to save the workflow as a new workflow.
```
Comment (abby-cyber) [inline docs-2.0/nebula-explorer/workflow/2.create-workflow.md:56]: and only modify  ---> and only make modifications

**Review by whitewum — APPROVED** [review]

**Review by abby-cyber — APPROVED** [review]

**Review by whitewum — APPROVED** [review]

**Pull request merged**